### DISCUSSION TITLE
File is too small

### DISCUSSION THREAD

**Discussion by Renanuya** [community/community#36830] [GitHub Education]
I need to upload proof of your academic status for student acc but IT SAYS FILE IS TO SMALL WHAT CAN I DO! pls help

**Comment by kude5n1k** [comment]
HI @Renanuya

Have you seen the info in this post: [Student Developer Pack Application & FAQs ➡️ Start Here](https://github.com/orgs/community/discussions/17814)

This blog (linked in that post) has some helpful information too: [How to get the GitHub Student Developer Pack without a student ID](https://github.blog/2019-07-30-how-to-get-the-github-student-developer-pack-without-a-student-id/)

**Reply by Renanuya** [reply]
Nope these are not helping

**Reply by kude5n1k** [reply]
Just try to increase the size of the image through any online service

**Reply by Renanuya** [reply]
İt's not working

**Reply by zephyrhours** [reply]
me too, the same trouble

**Reply by Renanuya** [reply]
> me too, the same trouble

Did you create a ticket?

**Reply by amdima2005** [reply]
> > me too, the same trouble
> 
> Did you create a ticket?

I did yesterday, no answer yet

**Reply by codewithgulam** [reply]
> I need to upload proof of your academic status for student acc but IT SAYS FILE IS TO SMALL WHAT CAN I DO! pls help


I was also faced this problem but I increased the image size online. It worked for me...

**Reply by kenny-orellana** [reply]
> > I need to upload proof of your academic status for student acc but IT SAYS FILE IS TO SMALL WHAT CAN I DO! pls help
> 
> I was also faced this problem but I increased the image size online. It worked for me...

Which size did you use?

**Reply by erajayky** [reply]
I was also facing same problem. I captured new photo using my smartphone. It worked for me

**Comment by abdullah-ctl** [comment]
Hello @Renanuya  

I am also facing the same problem but I fixed that by taking picture from my webcam.

**Reply by Renanuya** [reply]
But my webcam is 480p

**Reply by abdullah-ctl** [reply]
I think you should try

**Reply by Renanuya** [reply]
i will thank you

**Comment by PavelUdovichenko** [comment]
try to cut your pic by paint or smt like this

**Reply by Renanuya** [reply]
its not working

**Comment by dboyraz** [comment]
[Resizing ](https://www.adobe.com/express/feature/image/resize)for Standard > Portrait (2:3) solved it for me.

**Reply by Renanuya** [reply]
I will try

**Comment by ossanna16** [comment]
Thank you for asking a good question @Renanuya, I'm glad you found our community 🙂

**Reply by Renanuya** [reply]
^^

**Comment by AndreiZoltan** [comment]
I have the same problem.
No matter how I resize the picture, it still throws the error: "File is too small".

**Reply by Renanuya** [reply]
Yes i create a ticket but no one answered..

**Comment by codewithgulam** [comment]
It around 1mb

**Comment by DSDmark** [comment]
Hey @Renanuya,

I'm not sure if it could help you, but You could try this :- 

Go there [**image size increase**](https://www.imgonline.com.ua/eng/resize-image.php)

Select width and height 1920x1080 and in the bottom there is an **Output image format** section select **PNG-24 (without compression, with transparency support)**  click on **ok** Button your size will be increase.

**Reply by DSDmark** [reply]
I guess there are lots of people facing that problem https://github.com/community/community/discussions/37494

**Comment by dkhusted** [comment]
I know this is not an answear but minimum size for uploading is 4.5Mb. Dont know why its not written on their website...
